service/dap/server.go

```diff
@@ -1458,20 +1458,47 @@ func (s *Server) onAttachRequest(request *dap.AttachRequest) {
 // onNextRequest handles 'next' request.
 // This is a mandatory request to support.
 func (s *Server) onNextRequest(request *dap.NextRequest, asyncSetupDone chan struct{}) {
+	// All of the threads will be continued by this request, so we need to send
```

Comment:
how about a helper, so you don't repeat same code and comments 3 times?

Reply:
Done.